tls/s2n_security_policies.c

```diff
@@ -1349,7 +1360,7 @@ struct s2n_security_policy_selection security_policy_selection[] = {
     { .version = "20190122", .security_policy = &security_policy_20190122, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
     { .version = "20190801", .security_policy = &security_policy_20190801, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
     { .version = "20190802", .security_policy = &security_policy_20190802, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
-    { .version = "20200207", .security_policy = &security_policy_test_all_tls13, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
+    { .version = "20200207", .security_policy = &security_policy_20200207, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
```

Comment:
I wonder if we should commit the results of your `policy` tool on every security policy as a snapshot test, in order to avoid unintentional changes to random policies?

Reply:
I love that idea! https://github.com/aws/s2n-tls/issues/5306